### PR TITLE
community/repmgr bump to v4.0.2

### PR DESCRIPTION
**Pull request by blaggacao**
@jirutka Is that about how a version bump it should be done?

### CODE REVIEW

**Comment by blaggacao** [comment]
building the documentation seems to be now an extra step in version 4, so I removed it, as we would need extra dependencies... Not sure if that's the idea, but definitely reduces the package size, which is good in my personal opinion.

**Comment by blaggacao** [comment]
@jirutka See how the sample config, specially logging, has changed: https://github.com/2ndQuadrant/repmgr/blob/master/repmgr.conf.sample

File: community/repmgr/APKBUILD [hunk 1]
```diff
@@ -31,15 +31,12 @@ check() {
 package() {
 	cd "$builddir"
 
-	make USE_PGXS=1 DESTDIR="$pkgdir" install
+	./configure && make USE_PGXS=1 DESTDIR="$pkgdir" install
```
Comment (ncopa) [inline community/repmgr/APKBUILD]: do you really need to re-run configure before installing it? that does not seems right

Comment (blaggacao) [reply community/repmgr/APKBUILD]: I din't (don't) fully understand the build process. But I strongly guess it's not necesary.. :+1:

**Pull request closed without merging**